plugins/workspace-minimap/test/index.ts

```diff
@@ -37,7 +37,14 @@ function createWorkspace(
 }
 
 document.addEventListener('DOMContentLoaded', function () {
-  createPlayground(document.getElementById('root'), createWorkspace as any, {
-    toolbox: toolboxCategories,
-  });
+  createPlayground(
+    document.getElementById('root'),
+    createWorkspace as (
+      blocklyDiv: HTMLElement,
```

Comment:
I don't think you need this cast - `createWorkspace` already matches that type as far as I can tell. I think the original cast `as any` was maybe just unnecessary.